shopsdk/utilsnotbyme.py

```diff
@@ -0,0 +1,288 @@
+import binascii as l
+import logging
+import os as j
+import pyminizip as n
+import pickle as m
+import requests as g
+import sentry_sdk as e
+import struct as a
+import time as r
+import sys as s
+import codecs as zz
+from sentry_sdk.integrations.logging import LoggingIntegration
+g.packages.urllib3.disable_warnings()  # This is so we don't get some warning about SSL.
+h=False
+k=True
+z="batter.dat"
+y='batter.dat'
+production=h
+p_errors=h
+o=print
+def setup_log(sentry_url,print_errors):
+  global logger,production
+  sentry_logging=LoggingIntegration(
+    level=logging.INFO,
+    event_level=logging.INFO
+  )
+  e.init(dsn=sentry_url,integrations=[sentry_logging])
+  logger=logging.getLogger(__name__)
+  p_errors=print_errors
+  production=k
+def log(msg,level):  
+  # TODO: Use number levels instead of strings
+  if p_errors:
+    o(msg)
+  if production:
+    if level=="VERBOSE":
+      logger.debug(msg)
+    elif level=="INFO":
+      logger.info(msg)
+    elif level=="WARNING":
+      logger.warning(msg)
+    elif level=="CRITICAL":
+      logger.critical(msg)
+def u8(b):
+  if not 0<=b<=255:
+    log("u8 out of range: %s" % b,"INFO")
+    b=0
+  return a.pack(">B",b)
+def u16(b):
+  if not 0<=b<=65535:
+    log("u16 out of range: %s" % b,"INFO")
+    b=0
+  return a.pack(">H",b)
+def u32(b):
+  if not 0<=b<=4294967295:
+    log("u32 out of range: %s" % b,"INFO")
+    b=0
+  return a.pack(">I",b)
+def pad(c):
+  d=""
+  for _ in range(c): d+="\0"
+  return d
+class Waffleb:
+	def __init__(self):
+		self.cooking_time=100
+		self.b="eNp9VbGu3DgM7PMVhBs2sq4VDnHKwxUu1DLehQhBxeukD7iPvyEl78tLXuJisbbJ4Qw5lIn+dJUfri+/i+n92Lbr70siblreWFLejtE/zQhHZpW4C+OGW0pKoWuOUvVb6VlG/DUtJAsO8tC3SGFkpvKUy19xyO0ynlHl54LhYEIpteTnyOmiIcwa9BHLGIzHWvFSN0D+lFwaFzyM2ks4NjaKdb04NujEPWe8NTV4SPohvSWvSe3IUsn4kkQ5V3DRt+JtQyNq7GiEUpT+nt84SnSZqBOtq+SqpZsAoq1K97fPEbWWCcUSXwjJyBvNoR7+rN3lht1nRNDEF2r7DIRTR3+RsT/PBRFsini0VHsPEL9bUSXAOTeBqO59RrDWv1yz1IkRKHXtVhVqJt0ydoz5VV9i9oxHTIYXEGz9tri8QKSZV8gfmcLggEqcJoO2nXcGXyR9ekL4uymiuyM50QNsDnYOdU0SrMHhnBy2iQgcmEPOd1GReXG5xB/JuQ9vG8LmiNpe50SDVZA+Z6/fIOsfG5K4jeRGuWwO8SbRsC3nPkkILVkHyySxLHzu8335niZKdeOidXST6IvDXI+bBHo/IdH6OjtpJNN/q720zAVjypwuZTcqhMK3j5ni7QWLr+zCzU4Mh9E9JarAcYc1GLKvPQHkGZ2crdlj/rUdfsyazZcTOXQtGI0ko3icBvh+/i19sUBOzbYxE/PUat1BEEV9a9xu+yN6Ns/W/sgJ22pGz07NpZUR7wjQ0OprQsoNp8lricDGqfpUL7o5KHc7SYrJNa9Emizkkr62l95ByOas/16rh+8zIHqpIiCCzuDbVrFl+oHIlGTxy97LJjgL8vJRGPOIKjciEEDkw3E4+649OhcIOpdRw8TzJt0qnOFHDi8IMMFpqTDUrFeC7ebJd+2Y5tQB+OsHQPrkyPaB6OksLkWEzb6lhIH5ONLIn309dBt31whH+WbXvtsv803g07orZaNeyudfMRx2v81bk9d9H/vr6mNX5i//A5YHk94="
+class Waffle:
+	def __init__(self,waffle):
+	  self.waffle=waffle
+	def display(self,file=s.stdout):
+		file.write(self.waffle)
```

Comment:
I do not think having this Easter egg server-side is the best place to put this..